pandas/core/indexing.py

```diff
@@ -2023,6 +2023,16 @@ def ravel(i):
                         return ser._values.copy()
                     return ser.reindex(ax)._values
 
+        elif is_scalar(indexer) and isinstance(self.obj, ABCSeries):
```

Comment:
is_scalar case is always an integer right?  if so can use is_integer?

```
In [3]: ser = pd.Series([])
In [4]: self = ser.loc

In [7]: %timeit self.ndim == 1
65.6 ns ± 3.54 ns per loop (mean ± std. dev. of 7 runs, 10000000 loops each)

In [8]: %timeit isinstance(self.obj, ABCSeries)
340 ns ± 2 ns per loop (mean ± std. dev. of 7 runs, 1000000 loops each)
```

Reply:
Thanks, I somehow forget the ndim check ...